packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -73,27 +87,68 @@
         outline: none;
       }
     }
-
-    .is-disabled {
+    
+    .nav-link:disabled,
+    .nav-link.is-disabled {
+      background-color: var(--terra-paginator-nav-link-disabled-background-color, transparent);
+      border: var(--terra-paginator-nav-link-disabled-border, none);
+      box-shadow: var(--terra-paginator-nav-link-disabled-box-shadow);
       color: var(--terra-paginator-nav-link-disabled-color, rgba(28, 31, 33, 0.2));
       cursor: default;
-      pointer-events: none;
+      opacity: var(--terra-paginator-nav-link-disabled-opacity, 1);
+      pointer-events: none; // Prevent pointer interaction
 
       &:focus {
         box-shadow: var(--terra-paginator-nav-link-disabled-focus-box-shadow, none);
         outline: none;
       }
     }
 
-    .left-controls {
+    .nav-link.previous {
+      padding-left: var(--terra-paginator-nav-link-previous-padding-left, 0.5rem);
+
+      &.icon-only {
+        margin: 0;
```

Comment:
Do you think this needs to be theme-able to account for the issue below? 

![localhost_8080_(iPhone 5_SE) (6)](https://user-images.githubusercontent.com/10090859/61288655-3af51380-a7e5-11e9-800a-a1b660d6a535.png)

Reply:
Good catch. Missed a new variable. Adding back in.

Reply:
Added. https://github.com/cerner/terra-core/pull/2485/commits/86b3c05be3b0d754c50537bc6f864c4e6c9996ed